test/unit/option_sources/ids_params_test.rb

```diff
@@ -0,0 +1,30 @@
+require File.join(File.dirname(__FILE__), '../test_helper')
+
+describe HammerCLIForeman::OptionSources::IdsParams do
+  describe "get_options" do
+    class IdsParamsTestCommand < HammerCLIForeman::CreateCommand
+      resource :users
+      build_options
+    end
+
+    let(:cmd) { IdsParamsTestCommand.new("", { :adapter => :csv, :interactive => false }) }
+    let(:ids_params_source) { HammerCLIForeman::OptionSources::IdsParams.new(cmd) }
+
+    it "skips param when set" do
+      cmd.expects(:get_resource_ids).with { |res| res.name != :locations }.returns(nil)
```

Comment:
Nitpick: how come it works without `.at_least(0)` here? I'd expect using it in both tests or nowhere.